config/routes.rb

```diff
@@ -38,4 +39,16 @@
   post "/users", to: "users#create"
 
   get "/profile", to: "users#show"
+
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+
+  namespace :merchant do
+    get "/dashboard", to: "dashboard#show"
+  end
+  namespace :admin do
+    get "/dashboard", to: "dashboard#show"
+  end
+
+  delete "/logout", to: "sessions#destroy"
```

Comment:
since `logout` does several things, i'm not sold that this has to be a sessions#destroy. however, since that is included in the function, i can also see how it's not inappropriate.

Reply:
Maybe we can look up some examples on how to handle a user logout in Rails and reevaluate this route!

Reply:
I've also added this as a question for our check-in